frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./Footer.css";
+import Twitter from "./twitter.svg";
+import Copyright from "./copyright.svg";
```

Comment:
Can you  import your SVG pictures through "./public/pictures", instead of "import..from"?